routers/api/v1/repo/notes.go

```diff
@@ -78,7 +78,15 @@ func getNote(ctx *context.APIContext, identifier string) {
 		return
 	}
 
-	cmt, err := convert.ToCommit(ctx, ctx.Repo.Repository, ctx.Repo.GitRepo, note.Commit, nil, convert.ToCommitOptions{Stat: true})
+	verification := ctx.FormString("verification") == "" || ctx.FormBool("verification")
+	files := ctx.FormString("files") == "" || ctx.FormBool("files")
```

Comment:
Here and below, have these two new parameters been documented in swagger comment?

Reply:
Thanks! Fixed.